example/quantization/README.md

```diff
@@ -1,4 +1,248 @@
 # Model Quantization with Calibration Examples
+
+This folder contains examples of quantizing a FP32 model with Intel® MKL-DNN or CUDNN.
+
+<h2 id="0">Contents</h2>
+
+* [1. Model Quantization with Intel® MKL-DNN](#1)
+* [2. Model Quantization with CUDNN](#2)
+
+<h2 id="1">Model Quantization with Intel® MKL-DNN</h2>
+
+Intel® MKL-DNN supports quantization well with subgraph feature on Intel® CPU Platform and can bring huge performance improvement on Intel® Xeon® Scalable Platform. A new quantization script `imagenet_gen_qsym_mkldnn.py` has been designed to launch quantization for image-classification models with Intel® MKL-DNN. This script intergrates with [Gluon-CV modelzoo](https://gluon-cv.mxnet.io/model_zoo/classification.html) so that more pre-trained models can be downloaded from Gluon-CV and can be converted for quantization. This script also supports custom models.
```

Comment:
delete well and huge
feature-->features
improvement-->improvements on the Intel...
If you want to make performance comparisons and sugar it up with things like "huge", it's best to have something to compare it to rather than launch into hyperbolic statements.... just a suggestion.
I'm not sure at this point if I can get the best benefit from this because I'm not familiar with Intel® Xeon® Scalable Platform. Is that something that's everywhere? At all the cloud providers? Maybe link to more info on that?

no dash in image classification
integrates (spelling)
comma before "so"
I think it should be "downloaded... then converted" (instead of "and can")